src/sass/bootstrap-asu-extends.scss

```diff
@@ -1,66 +1,44 @@
-@import 'extends/card-image-and-content';
-@import '../components/cards/cards';
-@import 'extends/charts-and-graphs';
-@import 'extends/display-list';
-@import 'extends/dividers';
-@import 'extends/form-fields';
-@import '../components/form-overrides';
-@import '../components/form-fields/form-fields';
-@import 'extends/global-header';
-@import 'extends/globalfooter';
-@import '../components/global-footer/global-footer';
-@import '../components/grid-links/grid-links';
-@import '../components/headings/headings';
-@import '../components/heroes/heroes';
-@import '../components/icons/icons';
-@import 'extends/image-background-with-cta';
-@import 'extends/image-overlap';
+@import 'extends/icons';
+@import 'extends/images';
 @import 'extends/image-text-block';
-@import '../components/image/images';
+@import 'extends/image-overlap';
 @import 'extends/inset-box';
-@import '../components/layouts/layouts';
-@import '../components/lists/lists';
-@import 'extends/media_embed';
-@import 'extends/misc';
-@import '../components/modals/modals';
-@import '../components/page404/page404';
-@import '../components/pager/pager';
-@import 'extends/pagination';
-@import '../components/paragraphs/paragraphs';
+@import 'extends/cards';
+@import 'extends/list';
+@import 'extends/headings';
+@import 'extends/dividers';
+@import 'extends/pager';
+@import 'extends/tabbed-panels';
+@import 'extends/heroes';
+@import 'extends/breadcrumb';
+@import 'extends/sidebar';
+// @import 'extends/typography';
+@import 'extends/globalfooter';
+@import 'extends/blockquotes';
+@import 'extends/global-header';
+@import 'extends/form-fields';
 @import 'extends/quote-image-background';
-@import '../components/regions/regions';
-@import '../components/sidebar/sidebar';
-@import '../components/tables/tables';
-@import '../components/tabbed-panels/tabbed-panels';
+@import 'extends/tooltips';
+@import 'extends/video';
+@import 'extends/card-and-image';
+@import 'extends/grid-links';
+//@import 'extends/back-to-top-buttton';https://asudev.jira.com/browse/UDS-646
+@import 'extends/display-list';
+@import 'extends/charts-and-graphs';
+@import 'extends/anchor-menu';
+@import 'extends/card-image-and-content';
+@import 'extends/card-arrangements';
+@import 'extends/pagination';
+@import 'extends/modals';
+@import 'extends/image-background-with-cta';
 @import 'extends/tables';
-@import 'extends/tabs';
-@import '../components/tabs/tabs';
-@import 'extends/testimonials';
-@import '../components/tooltips/tooltips';
-@import 'extends/typography';
-@import '../components/text/text';
-@import '../components/video/video';
 @import 'extends/person-profile';
-
-// Misc
-@import '../components/ckeditor/ckeditor';
-@import '../components/testimonial-carousel/testimonial-carousel';
-@import '../components/story-hero/story-hero'; // override for image.
-@import '../components/anchors/anchors';
-@import '../components/cards/card-and-image-parallax';
-@import '../components/image-text-block/image-text-block';
```

Comment:
Imports like `@import '../components/image-text-block/image-text-block';` are going to be overrides for the CMS which we need to keep.